x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -175,7 +174,7 @@ export const createMetricThresholdExecutor = (libs: InfraBackendLibs) =>
         reason = alertResults
           .map((result) =>
             buildFiredAlertReason({
-              ...formatAlertResult(result[group], nextState === AlertStates.WARNING),
+              ...formatAlertResult(result[group] as any, nextState === AlertStates.WARNING),
```

Comment:
This is a temp fix. I was getting `spread types may only be created from object types`. Any tip on how to fix this one? @simianhacker